damus/Views/AddRelayView.swift

```diff
@@ -18,7 +18,7 @@ struct AddRelayView: View {
             Form {
                 Section("Add Relay") {
                     TextField("wss://some.relay.com", text: $relay)
-                        .autocorrectionDisabled(true)
+                        .disableAutocorrection(true)
```

Comment:
It seems disableAutocorrection is deprecated by Apple
https://developer.apple.com/documentation/swiftui/view/disableautocorrection(_:)